content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments -b pipeline-added
+$ cd get-started-experiments
+$ virtualenv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
+```
+
+### 📀 Get the data set
+
+The repository we cloned doesn't contain the dataset. In order to get the
+dataset we use `dvc pull` to update the missing data files. `dvc pull` is used
+in DVC repositories to update the missing data dependencies.
+
+```dvc
+$ dvc pull
+```
+
+Then we extract this file that contains labeled images.
+
+```dvc
+$ tar -xvzf data/images.tar.gz --directory data/
+```
+
+The repository already contains the necessary configuration to run the
+experiments. If you would like to learn how to configure a project to run DVC
+experiments, please refer to the last section.
+
+<details>
+
+## ⏲ Preparing a project for DVC experiments
+
+In this document we assume that there is already a configured DVC project to
+simplify the introduction. DVC experiments are a feature added in DVC 2.0 and
+requires a DVC pipeline defined in the project.
+
+Please refer to [install](/doc/install) if DVC is not installed in your system.
+
+If DVC is not initialized before in the project, you can do so by:
+
+```dvc
+$ dvc init
+```
+
+DVC also requires commands to be run and their dependencies to be defined as
+stages. We use `dvc stage add` to add a stage and set its dependencies.
+
+```dvc
+$ dvc stage add -n train \
+                -p model.conv_units \
+                -p train.epochs \
+                -d data/images \
+                -m metrics.json \
+                python3 src/train.py
+```
+
+The command tells DVC to create an experiment named `train`, and that for any
+change in `data/images/`, `model.conv_units` or `train.epochs`, we run an
+experiment using `src/train.py` that produces a new `metrics.json` file.
+
+Note that the parameters (added with `-p`) are in the default parameters file
+`params.yaml` and used in the code as normal, by reading the file. DVC only
+tracks the changes and updates them with `--set-param`. For other files that
+contain the parameters, DVC expects the filename to be provided similar to
+`-p myparams.yaml:epochs`.
+
+DVC is ready to run the experiments now!
+
+</details>
+
+## 👟 Running the experiment with default parameters
+
+The purpose of `dvc exp` subcommands is to run the pipeline for ephemeral
+experiments. By _ephemeral_ we mean the experiments can be run without
+committing parameter and dependency changes to Git. Instead the artifacts
+produced for each experiment are tracked by DVC and persisted on demand.
+
+Running the experiment with default project settings requires only the command:
+
+```dvc
+$ dvc exp run
+...
+Reproduced experiment(s): exp-7683f
+Experiment results have been applied to your workspace.
+...
+```
+
+It runs the specified command that writes the metrics values to `metrics.json`.
+
+<details>
+
+### 📜 If you used `dvc repro` before
```

Comment:
_ephemeral_ might make more sense in this section.

Reply:
It looks it's a sticky word to be avoided :) I kept this the same.